.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,24 @@
-##### SUMMARY
 <!--- Describe the change below, including rationale and design decisions -->
 
 <!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
 
-##### ISSUE TYPE
+### Type of pull request
 <!--- Pick one below and delete the rest -->
-- Bugfix Pull Request
```

Comment:
Bot will need updating to support old and new types and look under `Type of pull request`

Reply:
Yes, we mentioned that.